flex-checks/src/test/java/org/sonar/flex/checks/FunctionComplexityCheckTest.java

```diff
@@ -27,7 +27,8 @@
   @Test
   public void test() {
     FunctionComplexityCheck check = new FunctionComplexityCheck();
-    check.setMaximumFunctionComplexityThreshold(1);
+    // Function complexity can't be less than 1
+    check.setMaximumFunctionComplexityThreshold(2);
```

Comment:
@amelie-renard-sonarsource I would set `0` for the test to show that empty function has complexity of `1`.